libraries/plugins/xfpga/usrclk/fpga_user_clk.c

```diff
@@ -135,6 +135,8 @@
 #define IOPLL_WRITE_POLL_TIMEOUT_US   1000000 /* Write poll timeout */
 
 #define USRCLK_FEATURE_ID             0x14
+const char USRCLK_GUID[]              = "45AC5052C481412582380B8FF6C2F943";
```

Comment:
Define GUID and Feature ID to opae-sdk/blob/master/libraries/plugins/xfpga/types_int.h